apollo-router/src/plugins/telemetry/tracing/apollo.rs

```diff
@@ -28,7 +28,6 @@ impl TracingConfigurator for Config {
 
                 let exporter = apollo_telemetry::Exporter::builder()
                     .expose_trace_id_config(expose_trace_id.clone())
-                    .trace_config(trace_config.clone())
```

Comment:
why is this removed?

Reply:
It was unused. We stashed it in a struct, but never accessed it. Not sure why the compiler didn't pick it up.